filecoin-proofs/src/api/update.rs

```diff
@@ -27,17 +32,77 @@ use storage_proofs_update::{
 };
 
 use crate::{
-    api::util,
-    caches::{get_empty_sector_update_params, get_empty_sector_update_verifying_key},
+    api::util::{self, get_aggregate_target_len, pad_proofs_to_target},
+    caches::{
+        get_empty_sector_update_params, get_empty_sector_update_verifying_key, get_stacked_srs_key,
+        get_stacked_srs_verifier_key,
+    },
     chunk_iter::ChunkIterator,
     constants::{DefaultPieceDomain, DefaultPieceHasher},
     pieces::verify_pieces,
     types::{
-        Commitment, EmptySectorUpdateEncoded, EmptySectorUpdateProof, PieceInfo, PoRepConfig,
-        SectorUpdateConfig,
+        AggregateSnarkProof, Commitment, EmptySectorUpdateEncoded, EmptySectorUpdateProof,
+        PieceInfo, PoRepConfig, SectorUpdateConfig,
     },
 };
 
+#[derive(Debug, Clone)]
+pub struct SectorUpdateProofInputs {
+    //pub porep_config: PoRepConfig,
```

Comment:
It's not clear yet if it's needed coming from API.  The open question (@Kubuxu ?) is if it's possible to use inputs from a different porep config when aggregating sector update proofs.  There's otherwise no way to determine if the sectors are all the same size, same version, etc